test/test_stress.py

```diff
@@ -24,6 +24,9 @@ class TestStress(FacedancerTestCase):
     # - life-cycle ------------------------------------------------------------
 
     def setUp(self):
+        # set interface
+        self.set_interface(0, 0)
```

Comment:
This isn't available yet - it's only added in #122.

Reply:
If I change this to:
```suggestion
        self.device_handle.setInterfaceAltSetting(0, 0)
```

and similarly in `test_transfers.py`, then I can run the tests, but I still get failures of the form:
```
$ python -m unittest 
.F...
======================================================================
FAIL: test_control_out_transfer (test.test_transfers.TestTransfers.test_control_out_transfer)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/martin/repos/facedancer/test/test_transfers.py", line 101, in test_control_out_transfer
    self.check_out_transfer(length, data, bytes_sent)
  File "/home/martin/repos/facedancer/test/test_transfers.py", line 49, in check_out_transfer
    self.assertEqual(len(sent_data), len(received_data))
AssertionError: 258 != 194
```
with the `test.device` output reporting:
```
ERROR   | moondancer     | Discarding 64 bytes on control endpoint with no pending control request
```